activerecord/test/cases/tasks/database_tasks_test.rb

```diff
@@ -191,6 +207,23 @@ def test_creates_configurations_with_blank_hosts
         ActiveRecord::Tasks::DatabaseTasks.create_all
       end
     end
+
+    private
+
+      def with_stubbed_configuration_establish_connection
```

Comment:
It is unused in this PR. Can we remove this changes?

Reply:
Good catch. Removing.